src/fetch-blocks.ts

```diff
@@ -20,12 +20,14 @@ export const fetchAllBlocks = async (
   blocksConfig: BlocksConfig,
   blockApiUrl: string,
   lang: SupportedLang,
-  headers?: Headers
+  headers?: Headers,
+  styleVersion?: number
 ) => {
+  const blockView = styleVersion == 10 ? 'style10' : 'style9'
```

Comment:
I’m assuming the type coercion is by design. Is it to clever though? Easily missed by a tired developer on a Friday afternoon?